docs/reference/sql/functions/like-rlike.asciidoc

```diff
@@ -0,0 +1,71 @@
+[role="xpack"]
+[testenv="basic"]
+[[sql-like-rlike-operators]]
+=== LIKE and RLIKE Operators
+
+`LIKE` and `RLIKE` operators are commonly used to select data based on patterns. They usually act on a field placed on the left-hand side of
+the operator, but can also act on a constant (literal) expression. The right-hand side of the operator represents the pattern.
+Both can be used in the `WHERE` clause of the `SELECT` statement, but `LIKE` can also be used in other places, like defining indices list based
+on <<sql-index-patterns, index patterns>> or in combination with different <<sql-commands, SHOW commands>>.
+This section covers only the `SELECT ... WHERE ...` usage.
+
+IMPORTANT: Even though `LIKE` and `RLIKE` are valid options when searching or filtering in {es-sql}, full-text search predicates
+<<sql-functions-search, `MATCH` and `QUERY`>> are faster, more powerful and accurate and are the preferred alternative.
+
+NOTE: One significant difference between `LIKE`/`RLIKE` and the <<sql-functions-search, full-text search predicates>> is that the former
```

Comment:
I would extract this from a note and make it top-level section with a couple of examples:

[[sql-like-prefer-full-text]]
==== Prefer full-text search predicates

When using LIKE/RLIKE, do consider using <<sql-functions-search, full-text search predicates>> which are faster, much more powerful and accurate and offer the option of sorting by relevancy (results can be returned based on how well they matched). 

For example:
`WHERE foo LIKE 'bar' ` vs `WHERE MATCH(foo, 'bar')` 
`WHERE foo LIKE 'bar' AND tar LIKE 'goo' ` vs `WHERE MATCH('foo:bar AND tar:goo')` 

plus a couple of examples with boosting and fuzzyness.